package.json

```diff
@@ -30,6 +31,7 @@
         "logrocket": "^3.0.1",
         "pretty-bytes": "^6.1.0",
         "prismjs": "^1.29.0",
+        "svelte-i18n": "^3.6.0",
```

Comment:
We should discuss if this is the best library for the console. @TorstenDittmann has some features he'd like, and it also seems unmaintained.